sparkpost/transmissions.py

```diff
@@ -196,6 +196,11 @@ def send(self, **kwargs):
         results = self.request('POST', self.uri, data=json.dumps(payload))
         return results
 
+    def _fetch_get(self, transmission_id):
```

Comment:
To support async transports fetch methods should only return whatever they fetched - in async it will be a future, not a real value that can be manipulated. It enables code in sparkpost/tornado/transmissions.py